cernopendata/modules/theme/assets/semantic-ui/scss/visualise.scss

```diff
@@ -1,49 +1,74 @@
-#vis-histograms {
-  color: #606D75;
-}
+$histograms-text: #606d75;
+$histograms-dropdown: #6c757d;
+$histograms-button: #6c757d;
```

Comment:
I'd name these variables to describe the color itself as we have in `variables.scss`. Unless one is able to read HEX, they won't tell you much.

PS. `$histograms-dropdown` and `$histograms-button` have the same value.

Reply:
Since all these colors were different variants of grey I took functional naming approach, but after playing a bit more with these I've reduced the colors to s single grey color, because the differences were very minimal